packages/core/stories/spinner/spinner.stories.tsx

```diff
@@ -41,6 +50,79 @@ Large.args = {
   size: "large",
 };
 
+export const Loading: Story = () => {
+  const [exampleIsLoading, setExampleIsLoading] = useState(true);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setExampleIsLoading(false);
+    }, 5000);
+  });
+
+  const handleClick = () => {
+    setExampleIsLoading(true);
+  }
+
+  return (
+      <div style={{ textAlign: "center" }}>
+        <p>Please wait for action to complete.</p>
+        {exampleIsLoading ? (
+          <Spinner
+            style={{ margin: "auto" }}
+            aria-label="Panel is loading"
+            size="large"
+          />
+        ) : (<>
+          <p>Action complete.</p>
+          <Button onClick={handleClick}>Reload</Button>
+          </>
+        )}
+      </div>
+  );
+};
+
+export const PartialLoading: Story = () => {
```

Comment:
I found this example very unstable (only tested on Mac Safari so far) on storybook preview, where sometimes the spinning shows up after clicking the button, sometimes not / or a lot shorter when using button -> reset -> button -> reset a few times in a row.

Reply:
This useEffect needs to be updated to have a dependency array as well as a clean-up function.

Reply:
Updated, lmk if that's okay!

Reply:
tyvm @joshwooding that's great